### DISCUSSION TITLE
How to remove the MSP version error when compiling sky motes in Contiki Cooja

### DISCUSSION THREAD

**Discussion by rabailmushtaq** [contiki-ng/contiki-ng#1625] [Q&A]
I'm working on Contiki-ng and I have MSPGCC4.7.2 version installed. But every time I try to compile a sky mote, it shows me this error.

![9mHiS](https://user-images.githubusercontent.com/88725712/129687401-26bbfe38-eaba-427b-8da0-6ec1bc12c0d1.png)

I have tried re-installing it multiple times but the error is still there.

**Comment by joakimeriksson** [comment]
Did you submit the correct screenshot? I. do not see a specific error - just the border-router starting and getting a prefix.

**Reply by rabailmushtaq** [reply]
Sorry, just updated the picture.

**Reply by joakimeriksson** [reply]
What happens if you compile outside of COOJA for sky?

**Reply by rabailmushtaq** [reply]
I'm not sure how we can compile sky motes outside of Cooja

**Comment by pjonsson** [comment]
The `msp430-gcc` found by Cooja is not the version 4.7.2 you have installed. Add `V=1` to your make command, so `make V=1 hello-world.sky TARGET=sky` and you will see which msp430-gcc Cooja is using.

Unless you want the older version of msp430-gcc for something else, it is probably easiest to uninstall your Linux distribution's msp430-gcc package.
